index.html

```diff
@@ -0,0 +1,198 @@
+             </header>
+        </div>
+    </div>
+
+       <!-- navbar section start from here -->
+     <div class="contianer-fluid nav-bar-fluid-container">
+        <div class="container">
+             <nav class="row nav-bar-container">
+                <div class="col-2"> <img class="nav-logo" src="./images/educationlogo1.png" alt="courselogo"></div>
+                <div class="col-10 d-flex justify-content-end"> 
+                    <ul class="nav-list">
+                        <li><a href="./about.html"> About</a></li>
+                        <li><a id="coursesiconindex" href="#main-courses-section"> Courses</a></li>
+                        <li><a id="joiniconindex" href="#"> Join</a></li>
+                        <li><a id="sponsericonindex" href="#"> Sponser</a></li>
+                        <li><a id="latesticonindex" href="#"> Latest</a></li>
+                        <li><a href="#"><button class="nav-bar-btn">Start Now!</button></a></li>
+                    </ul>
+                </div>
+        
+             
+             </nav>
+        </div>
+     </div>
+
+     <!-- Main home page section start from here -->
+    <div class="contianer-fluid home-page-fluid-container">
+        <div class="container-fluid menu-bar-fluid">
+            <div class="row">
+                <div id="wrap" class="wrap">
+                    <div id="bar1" class="menu"></div>
+                    <div id="bar2" class="menu"></div>
+                    <div id="bar3" class="menu"></div>
+                </div>
+            </div>
+            <div id="navlistdisplay1" class="row">
+                <ul class="nav-list-mobile">
+                    <li><a href="./about.html"> About</a></li>
+                    <li><a id="coursesiconmobile" href="#main-courses-section">Main Courses</a></li>
+                    <li><a id="joiniconmobile" href="#"> Join</a></li>
+                    <li><a id="sponsericonmobile" href="#"> Sponser</a></li>
+                    <li><a id="latesticonmobile" href="#"> Latest</a></li>
+                </ul>
+            </div>
+        </div>
+
+      
+
+        <div class="container main-home-container">
+            <div class="row heading-1">
+                <h1>"Welcome, Students!"</h1>
+            </div>
+            <div class="row heading-2">
+                <h2>WEB DEVELOPMENT COURSE BATCH<br> 2023</h2>
+            </div>
+            <div class="row para-heading">
+                <p>A joyful celebration is going to start soon By the opening of classes. Thereby, creating a change in education environment by giving admission to students from 180 countries.</p>
+            </div>
+            <div class="row heading-3">
+                <h3>2023.10.15(THU) ~ (FRI)</h3>
+            </div>
+            <div class="row heading-4">
+                <h4>@Webly-Education Pakistan, And many more</h4>
+            </div>
```

Comment:
- [ ] Please this part of your project is not responsive to a media screen of width `375px` so make sure you fix it according to the [given design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).
![image](https://user-images.githubusercontent.com/92303891/218154625-1fcf6c44-27fd-4d27-8b6f-177810391398.png)

Reply:
![image](https://user-images.githubusercontent.com/116083031/218162796-a3fd988d-cc8b-4258-96c2-a83c3783afb1.png)
Please kindly see its responsive at this width